HTTPlayground/base.py

```diff
@@ -13,32 +26,50 @@ def send_headers(request, response_code: int = 200, content_type: str = 'text/pl
     request.end_headers()
 
 
+def process_static(request):
+    filename = request.path.rsplit('/', 1)[-1]
+    path = Path(BASE_DIR, STATIC_URL, filename)
+    if path.exists():
+        send_headers(request, content_type=guess_type(request.path)[0])
+        return path.read_text('utf-8')
+
+
 class CookieHandler:
     @staticmethod
-    def is_cookie(request, cookie_pair: dict):
+    def has_cookie(request, *, name, value):
         raw_cookies = request.headers.get('Cookie')
         cookies = SimpleCookie(raw_cookies)
-        k, v = tuple(cookie_pair.items())[0]
-        cookie = cookies.get(f'{k}')
-        return getattr(cookie, 'value', None) == v
+        cookie = cookies.get(name)
+        return getattr(cookie, 'value', None) == value
+
+    @staticmethod
+    def generate_cookie_for_remove(request, *, name):
+        cookie = SimpleCookie(request.headers.get('Cookie'))
+        cookie[name]['path'] = '/'
+        cookie[name]['expires'] = 'Thu, 01 Jan 1970 00:00:00 GMT'
+        cookie[name]['httponly'] = True
+        return cookie.output(header='', sep='')
 
     @classmethod
-    def generate_cookie(cls, request, cookie_pair: dict):
+    def generate_cookie(cls, request, *, name, value):
         cookie = SimpleCookie(request.headers.get('Cookie'))
-        k, v = tuple(cookie_pair.items())[0]
-        cookie[f"{k}"] = v
-        cookie[f"{k}"]['path'] = '/'
-        cookie[f"{k}"]['httponly'] = True
+        cookie[name] = value
+        cookie[name]['path'] = '/'
+        cookie[name]['httponly'] = True
         return cookie.output(header='', sep='')
 
 
-class FileReader:
-    @staticmethod
-    def read(filename: str, mode: str = 't', encoding: str = 'utf-8'):
-        """binary mode - b, text mode - t"""
-        is_template = filename.endswith('.html')
-        path = Path(BASE_DIR, (TEMPLATE_DIR if is_template else STATIC_URL), filename)
-        if path.exists():
-            return path.read_text(encoding) if mode == 't' else path.read_bytes()
-        else:
-            raise FileNotFoundError(f'Cannot find file:\n{path}')
+class TokensDB:
```

Comment:
1. I'm not sure why you need a token db at all. You can calculate the current hash(cookie+salt) and compare it with the form token value without any dbs at all
1. `classmethod` will not help you here in many cases: multiprocess server, server could be reloaded between the form load and send, etc. So in multi-request cases it's better to use some more persistent storage like redis, db etc. Generally speaking you can rely on python memory only within a single request-response lifetime.